### PR TITLE
GC-567 - Implement Reactor API.

### PR DESCRIPTION
**Pull request by ramonPaulAlvarez**

### CODE REVIEW

File: src/evrythng/entities/reactor.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+        'script': 'str',
+        'files': 'dict',
+        'manifest': 'ref',
+    },
+    'readonly': ('id', 'createdAt', 'updatedAt'),
+    'writable': ('type', 'script', 'files', 'manifest'),
+    'required': (),
+}
+
+reactor_log_field_specs = {
```
Comment (ramonPaulAlvarez) [inline src/evrythng/entities/reactor.py:20]: I wanted to validate the log entries.

**Comment by codeadict** [comment]
LGTM CR:+1:

**Comment by codeadict** [comment]
You should define a new version in setup.py so we can push it to pypi

**Comment by LyleScott** [comment]
CR+1

**Pull request closed without merging**